### PR TITLE
aa user my comparisons have svgs

### PR DESCRIPTION
**Pull request by juskek**
image

image

### CODE REVIEW

File: src/utils/helpers/getEmissionComparison.ts [hunk 1]
```diff
@@ -8,20 +11,38 @@ const formatComparison = (emission: number, comparison: string): string => {
...
-        "% " +
-        "Google Search"
-    );
+    return {
+        description:
+            ((emissions / REAL_LIFE_EXAMPLES["Google Search"]) * 100).toFixed(
+                0
+            ) +
+            "% " +
+            "Google Search",
```
Comment (RSAMUEL12) [inline src/utils/helpers/getEmissionComparison.ts:45]: change this to a string literal

File: src/assets/plastic-bottle.svg
Comment (Lerri-Cofannos) [inline src/assets/plastic-bottle.svg:1]: Wasn't there an image that has a similar style to the coffee cup? Or the other way around?

File: src/utils/helpers/getEmissionComparison.ts (hunk 1 above)
Comment (RSAMUEL12) [reply src/utils/helpers/getEmissionComparison.ts:45]: The description value I mean

File: src/view/popup/pages/ResultsPage.tsx [hunk 2]
```diff
@@ -46,6 +46,7 @@ export const ResultsPage = ({
             <h1 className="text-2xl font-bold text-center">
                 Sustainability Calculator
             </h1>
+
```
Comment (Lerri-Cofannos) [inline src/view/popup/pages/ResultsPage.tsx:49]: An empty line is not pretty to see...

Comment (juskek) [reply src/view/popup/pages/ResultsPage.tsx]: fixed!

File: src/utils/helpers/getEmissionComparison.ts (hunk 1 above)
Comment (juskek) [reply src/utils/helpers/getEmissionComparison.ts]: done!

**Review by Lerri-Cofannos — APPROVED** [review]

**Pull request merged**